src/tests/ftest/util/dmg_utils.py

```diff
@@ -44,72 +44,270 @@ def __init__(self, path):
         self.debug = FormattedParameter("-d", False)
         self.json = FormattedParameter("-j", False)
 
-    def get_action_command(self):
-        """Get the action command object based on the yaml provided value."""
-        # pylint: disable=redefined-variable-type
-        if self.action.value == "format":
-            self.action_command = self.DmgFormatSubCommand()
-        elif self.action.value == "prepare":
-            self.action_command = self.DmgPrepareSubCommand()
-        elif self.action.value == "create":
-            self.action_command = self.DmgCreateSubCommand()
-        elif self.action.value == "destroy":
-            self.action_command = self.DmgDestroySubCommand()
+    def get_sub_command_class(self):
+        """Get the dmg sub command object based upon the sub-command."""
+        if self.sub_command.value == "network":
+            self.sub_command_class = self.NetworkSubCommand()
+        elif self.sub_command.value == "pool":
+            self.sub_command_class = self.PoolSubCommand()
```

Comment:
(pylint-redefined-variable-type) Redefinition of self.sub_command_class type from util.dmg_utils.DmgCommand.NetworkSubCommand to util.dmg_utils.DmgCommand.PoolSubCommand